.github/workflows/golangci-lint.yml

```
@@ -17,6 +17,10 @@ jobs:
         working-directory:
         - ""
         - api
+        - hack/tools/release
+        include:
+          - working-directory: hack/tools/release
+            additional-args: --build-tags=tools --modules-download-mode=readonly
```

```suggestion
        - working-directory: hack/tools/release
          additional-args: --build-tags=tools --modules-download-mode=readonly
```
All YAML is indented in k8s style.